client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -59,15 +62,18 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             }}
             name="email"
             autoComplete="email"
+            placeholder="Your email"
             autoFocus
             helperText={touched.email ? errors.email : ''}
             error={touched.email && Boolean(errors.email)}
             value={values.email}
             onChange={handleChange}
           />
+          <Typography fontSize={10} fontWeight={'bolder'} className={classes.label}>
+            PASSWORD
```

Comment:
The text transform should be used for uppercase. This is for accessibility, screen readers read these out as abbreviations.